submissions/Valkirin/tiny-js-world/index.js

```diff
@@ -0,0 +1,50 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+"use strict";
+
+class Inhabitant {
+  constructor(type, name, legs, hands, say) {
+    this.type = type;
+    this.name = name;
+    this.legs = legs;
+    this.hands = hands;
+    this.say = say;
+  }
+}
+
+// ======== OBJECTS DEFINITIONS ========
+
+const dog = new Inhabitant("dog", "Rex", 4, 0, '"Woff!"');
+
+const cat = new Inhabitant("cat", "Buba", 4, 0, '"Meow!"');
+
+const woman = new Inhabitant(
+  "woman",
+  "Katalin",
+  2,
+  2,
+  '"need make up my nails"'
+);
+
+const man = new Inhabitant(
+  "man",
+  "Bob",
+  2,
+  2,
+  '"Everything will be done. But tomorrow."'
+);
+
+const joinMethod = function(arg) {
```

Comment:
You [could ](https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/)use here ES6 arrow function, i guess it will add more readability.

Reply:
Ok. I'm resolved it